src/api/authorData.js

```diff
@@ -1,25 +1,71 @@
-// import axios from 'axios';
-// import firebaseConfig from './apiKeys';
+import axios from 'axios';
+import firebaseConfig from './apiKeys';
 
-// const dbUrl = firebaseConfig.databaseURL;
+const dbUrl = firebaseConfig.databaseURL;
 
 // FIXME:  GET ALL AUTHORS
-const getAuthors = () => {};
+const getAuthors = () => new Promise((resolve, reject) => {
+  axios.get(`${dbUrl}/authors.json`)
```

Comment:
```suggestion
const getAuthors = (uid) => new Promise((resolve, reject) => {
  axios.get(`${dbUrl}/authors.json?orderBy="uid"&equalTo="${uid}"`)
```

and be sure to pass uid when `getAuthors(uid)` is called